helper-cli/src/main/kotlin/utils/Extensions.kt

```diff
@@ -219,8 +219,8 @@ internal fun OrtResult.getViolatedRulesByLicense(
 /**
  * Return the [Provenance] of the first scan result matching the given [id] or null if there is no match.
  */
-internal fun OrtResult.getScannedProvenance(id: Identifier): KnownProvenance? =
-    getScanResultsForId(id).firstNotNullOfOrNull { it.provenance as? KnownProvenance }
+internal fun OrtResult.getScannedProvenance(id: Identifier): RemoteProvenance? =
```

Comment:
Commit message:
- typo in "functins"
- "should be cleaner"... *should* be or *is*? 😉 (More clearly, this sounds as if you don't know what you're doing, so use "is".)

Reply:
Updated the commit message.